src/pages/home/report/ReportActionItem.js

```diff
@@ -121,6 +180,24 @@ class ReportActionItem extends Component {
         this.setState({isPopoverVisible: false});
     }
 
+    /**
+     * Used to calculate the Context Menu Dimensions
+     *
+     * @returns {JSX}
+     * @memberof ReportActionItem
+     */
+    measureContent() {
+        return (
+            <ReportActionContextMenu
+                isVisible
+                selection={this.selection}
+                reportID={this.props.reportID}
+                reportAction={this.props.action}
+                hidePopover={this.hidePopover}
+            />
+        );
+    }
+
```

Comment:
Confused with the naming of this, it says it measures the content but it just returns a react component?

Reply:
It could be `measureContentPlaceholder`.

Reply:
Ah nevermind, I didn't understand what ReportActionContextMenu did but seeing how it's used makes more sense